### PR TITLE
add name to migrate status response (close #2376)

### LINKED ISSUE #2376: Add names to `hasura migrate status` command output
The output for this command currently lists the version, database and source status. It would be nice if it showed the name of migration as well.

### PR DESCRIPTION
**Pull request by eulercamposbarros**
### Description

Adds Identifier to CLI status command response.

### Affected components 

- CLI

### Related Issues

#2376 

### Solution and Design

The source Driver interface was extended to expose the Identifier. 

### Steps to test and verify

Before:
image

After:
image

### Limitations, known bugs & workarounds

Should the IDENTIFIER column be on the rightmost place (see image above)? Maybe it should be placed next to VERSION.

### CODE REVIEW

**Comment by eulercamposbarros** [comment]
Sorry for the mess with all these commits. If this gets accepted, I suggest doing a squash merge to avoid clutter the git master's history.

**Comment by marionschleifer** [comment]
@eulercamposbarros could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

File: cli/commands/migrate_status.go [hunk 1]
```diff
@@ -74,12 +74,13 @@ func printStatus(status *migrate.Status) *bytes.Buffer {
 	buf := &bytes.Buffer{}
 	out.Init(buf, 0, 8, 2, ' ', 0)
 	w := util.NewPrefixWriter(out)
-	w.Write(util.LEVEL_0, "VERSION\tSOURCE STATUS\tDATABASE STATUS\n")
+	w.Write(util.LEVEL_0, "VERSION\tSOURCE STATUS\tDATABASE STATUS\tIDENTIFIER\n")
```
Comment (arvi3411301) [inline cli/commands/migrate_status.go]: @eulercamposbarros Can you change `IDENTIFIER` to `NAME`.

Also, change the order to `VERSION    NAME    SOURCE STATUS    DATABASE STATUS`

Comment (eulercamposbarros) [reply cli/commands/migrate_status.go]: Done!

**Pull request merged**